orchestra/workflow/load.py

```diff
@@ -128,11 +128,11 @@ def load_workflow_version(version_data, workflow, force=False):
             .filter(workflow_version=version)
             .values_list('slug', flat=True)
         )
-        if new_step_slugs != old_step_slugs:
-            raise WorkflowError('Even with --force, you cannot change the '
-                                'steps of a workflow. Drop and recreate the '
-                                'database to reset, or create a new version '
-                                'for your workflow.')
+        if old_step_slugs - new_step_slugs:
```

Comment:
the critical change: only checks that `new_step_slugs` isn't missing anything that was in `old_step_slugs`.

Reply:
In the PR description, you made an argument that we should only do this for leaf nodes, but here theoretically i can insert a node in between two others, no? Or does some logic up above prevent that?

Reply:
Subsequent logic (l.184) checks that no dependencies have changed, so you can't update the step that would come 'after' to say `creation_depends_on.append(my_new_step)`.

If you're thinking about the submission dependency graph, you can insert a node into the graph by saying that your new step has `submission_depends_on = some_existing_step`. However, that doesn't feel risky because it only affects whether or not you can submit your new task, it doesn't affect other tasks. (This script doesn't even verify the submission constraint topology, so you can mess with it at will anyways :-) ).